system/doc/reference_manual/typespec.xml

```diff
@@ -83,17 +83,30 @@
     </p>
     <pre>  atom() | integer()</pre>
     <p>
-      Because of subtype relations that exist between types, types
+      Because of subtype relations that exist between types, all types,
+      except <c>dynamic()</c>,
       form a lattice where the top-most element, <c>any()</c>, denotes
       the set of all Erlang terms and the bottom-most element, <c>none()</c>,
       denotes the empty set of terms.
     </p>
+    <p>
+      To facilitate <url href="https://en.wikipedia.org/wiki/Gradual_typing">
+      gradual typing</url> of Erlang, the type `dynamic()` is provided.
+      It is similar to
+      <url href="https://docs.python.org/3/library/typing.html#the-any-type">Any</url>
+      in Python,
+      <url href="https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#any">any</url>
+      in TypeScript and
+      <url href="https://docs.hhvm.com/hack/built-in-types/dynamic">dynamic</url>
+      in Hack.
```

Comment:
Should a note be added that Dialyzer "treats dynamic the same as any/term" (there is probably a better way to phrase this) but other type systems running on the BEAM may rely on its distinctive meaning?

Reply:
Thanks for pointing this out. - Yes, it is worth to clarify it explicitly.

Reply:
Clarified in 79697d086ffd6972453cb1c17920280e5a2616b1